alert-database/src/main/resources/liquibase/7.1.0/client-certificates-tables.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <changeSet author="mau" id="create-client-certificates-table">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists schemaName="alert" tableName="client_certificates"/>
+            </not>
+        </preConditions>
+        <createTable schemaName="alert" tableName="client_certificates">
+            <column name="id" type="UUID" defaultValueComputed="uuid_generate_v4()">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="alias" type="VARCHAR(128)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="private_key_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="certificate_content" type="VARCHAR"/>
```

Comment:
I don't believe we want this to be nullable either.  See if you can add the constraint nullable=false for the content field as well.